pallets/relayer/src/lib.rs

```diff
@@ -452,21 +453,6 @@ impl<T: Config> Module<T> {
         Ok(())
     }
 
-    fn ensure_pallet_is_subsidised(pallet: &[u8]) -> Result<Option<()>, InvalidTransaction> {
-        match pallet {
-            // These pallets are subsidised by the paying key.
-            b"Asset" | b"ComplianceManager" | b"CorporateAction" | b"ExternalAgents"
-            | b"Permissions" | b"Portfolio" | b"Settlement" | b"Statistics" | b"Sto"
```

Comment:
Note that the `Permissions` pallet doesn't have any extrinsics, so it didn't even need to be in this list.